RecoLocalCalo/HcalRecAlgos/src/parseHFPhase1AlgoDescription.cc

```diff
@@ -30,6 +30,12 @@ parseHFPhase1AlgoDescription(const edm::ParameterSet& ps)
         const bool rejectAllFailures =
             ps.getParameter<bool>("rejectAllFailures");
 
+        float minChargeForUndershoot = -10000.f, minChargeForOvershoot = -10000.f;
+        if (ps.existsAs<double>("minChargeForUndershoot"))
```

Comment:
I do not find this remote similarity to be a sufficient argument.
As it stands now, there are no alternative plugins/implementations of parseHFPhase1AlgoDescription relevant to this parameter.
The "plugin" argument applies mostly only to the old code where the plugin infrastructure is built up and is complex enough to allow an exception.